WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -162,13 +162,13 @@ public async Task<IActionResult> GetTransactionsAsync([FromQuery, Required] IEnu
 			{
 				var hexes = new Dictionary<uint256, string>();
 				var queryRpc = false;
-				IRPCClient batchingRpc = null;
-				List<Task<Transaction>> tasks = null;
+				IRPCClient? batchingRpc = RpcClient.PrepareBatch();
+				List<Task<Transaction>>? tasks = new();
```

Comment:
Why are they nullable?

Reply:
Oops, i left the ? there, they don't need it.